inc/functions/notification.php

```diff
@@ -18,7 +18,7 @@
  * @param  array $data Notification data.
  * @return \WP_Error | true
  */
-function notification( $data = [] ) {
+function notification( $data = array() ) {
```

Comment:
Bad replacement, we use short syntax only

Reply:
Will stay like this, due to phpcs problems